src/content/protocols/compound-v2.md

```diff
@@ -0,0 +1,338 @@
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| GovernorBravoDelegate                       | \_acceptAdmin                   | Moves pendingAdmin into admin and clears pendingAdmin. This finalizes the transfer of all governance privileges (voting delays, periods, thresholds, guardianships, etc.) to the new admin. A compromised or malicious pendingAdmin could call this and immediately seize full governance control—reconfiguring parameters or blocking proposals with no recourse.                                 | pendingAdmin                        |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+
+## Dependencies
+
+The compound-v2 protocol relies on a Chainlink oracle feed to price collateral and base assets in the system. It doesn’t have a full secondary oracle feed or liveness checks, but its **`PriceOracle`** contract does include a minimal “fallback” feature:
+
+-    **Fixed‐price fallback**: if an administrator has set a fixedPrice for a given asset, then getUnderlyingPrice will return this hard-coded price in case of error on the primary price source.
+
+-   **Impact**: this allows the protocol to continue operating with a sane collateral price if the main feed is down or misbehaving, preventing total system paralysis.
+
+So the protocol does not validate asset prices returned by Chainlink or offer a valid fallback oracle mechanism.
+
+**`PiceOracle`** fetches prices from Chainlink Price Feeds when requested for a specific cToken.
+
+The Chainlink oracle system itself is upgradeable without decentralized ownership over those permissions. This dependency thus introduces centralization risk in the Compound-v2 protocol.
+
+The [Compound community multisig](https://etherscan.io/address/0xbbf3f1421d886e9b2c5d716b5192ac998af2012c) has the ability to update the configs on the Price Feed. The multisig has the flexibility to make the following changes:
+
+-   **Add new markets**
+
+-   **Update price feed for markets**
+
+-   **Update fixed price for markets**
+
+-   **Remove old markets**
+
```

Comment:
Please try and avoid the bullet points again, just simple and straightforward paragraphs, what can be done and what are the consequences. Please also include a small paragraph on why Chainlink is a centralized dependency, you can take the 2nd paragraph from Compound V3's Dependencies section.